nixos/lib/make-disk-image.nix

```diff
@@ -28,6 +28,9 @@
   #   partition of reasonable size is created in addition to the root partition.
   # For "legacy", the msdos partition table is used and a single large root
   #   partition is created.
+  # For "legacy+gpt", the GPT partition table is used, a 1049K no-fs partition for
```

Comment:
1049K? I see 1MiB later on.

I would also document that it's a [BIOS Boot partition](https://en.wikipedia.org/wiki/BIOS_boot_partition), the actual human-friendly name of the partition type that parted calls "bios_grub".

Reply:
Yes, I was surprised by this too.

Check out out what parted has to say:

```
Disk /build/nixos.raw: 2147MB
Sector size (logical/physical): 512B/512B
Partition Table: gpt
Disk Flags:

Number  Start   End     Size    File system  Name     Flags
 1      1049kB  2097kB  1049kB               no-fs    bios_grub
 2      2097kB  2146MB  2144MB  ext4         primary
```

Reply:
from sfdisk:

```>>> print

Device     Start         End     Sectors  Size Type
/dev/xvda1  2048        4095        2048    1M BIOS boot
/dev/xvda2  4096 10485759966 10485755871  4.9T Linux filesystem
```